### ISSUE TITLE
Label Screen Walkthrough

### ISSUE THREAD

**Issue by Abby-Wheelis** [e-mission/e-mission-docs#953]
At one point, there was a walkthrough in the Label screen that showed users what the features did / how to use them. Since this was in angular, it isn't a part of the migrated screen. Filing this issue for consideration of adding an on-demand walkthrough or hints that appear conditionally when the user first encounters something (like a popup about confirming inferred labels the first time they occur)

**Comment by Abby-Wheelis** [comment]
We have some user feedback indicating that the purpose of the "checkmark" isn't very clear, or that they didn't realize at first that it confirmed inferred labels with one click

We're also somewhat concerned that user's might not know what draft trips are or how they work, which came up when realizing the text "This trip has not yet been analysed. If it stays in this state, please ask your sysadmin to check what is wrong."

Both of these places are candidates for the new hints system.

**Comment by shankari** [comment]
Additional feedback on the draft trip:
- is this green? it looks like a gray/green
   - if you have red/green color blindness, not sure what color it is
- not the same shade as the old draft trips
   - old trips used to have the word draft in them, which was helpful
- green does not connote draft to me
   - usually green has positive connotation
   - (n.b. from shankari, the original draft trips were green because the e-mission theme color was green
   - then this should be light/washed out blue
- end point icons are missing - is that also deliberate to indicate that it is a draft?
   - if you want us to label draft trips, you must show the icons
   - because without that, the only way to remember the purpose of the trip is by looking at the timestamps
   - one end is home and the other is some place in the next town; but was I going there or coming back?

**Comment by shankari** [comment]
positive feedback:
- the flag icon is very good to represent the end
- the pinpoint icon is not that good to represent the start, but you only need one to be intuitive

- having different colors for the start and end icons as before was very helpful
   - red and green were good even with mild red/green blindness
   - but red/green blindness aware colors would be bettter

- confirm button between the mode and purpose is bad - too easy to fat finger it and go to the detail screen instead
    - this is the most improvement because it wastes time
    - oh they got it right, I am going to click to confirm. aaaah! now I have to go back from the detail screen
    - single biggest drain on productivity as labeler

**Comment by JGreenlee** [comment]
> * old trips used to have the word draft in them, which was helpful

Yes, they should definitely have some kind of indicator (and possibly description)

>   * usually green has positive connotation
>   * (n.b. from shankari, the original draft trips were green because the e-mission theme color was green
>   * then this should be light/washed out blue

I think grey makes sense here. I didn't know the reason they were green and thought it was weird, so I made them greyer.
We can experiment with a grey that is slightly bluish.
I don't recommend a lighter tone of blue because it would be too similar to Place cards (in configurations that actually show places)

> * end point icons are missing - is that also deliberate to indicate that it is a draft?

Not deliberate. Missing (but only sometimes?)

> * having different colors for the start and end icons as before was very helpful

Working on tweaking the icons for next release. Let us know what you think then

> * too easy to fat finger it and go to the detail screen instead

Definitely. At the very least, for now we can make the clickable area bigger.
I'd love to somehow gracefully incorporate it with the buttons so that its purpose is clear. Right now it's just a floating checkmark and not necessarily intuitive. The perfect idea has not struck me but I'll keep pondering on it.
